src/components/Home.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import 'react-testing-library/cleanup-after-each';
+import { render, cleanup } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import Home from './Home';
+
+afterEach(cleanup);
+
+test('if the home component rendered', () => {
+  const component = render(<Home />);
+  expect(component).toBeTruthy();
```

Comment:
hi @okoroemeka what does this 'toBeTruthy' test?

Reply:
Resolved.